diff --git a/dist/build/webpack/loaders/next-flight-loader/index.js b/dist/build/webpack/loaders/next-flight-loader/index.js
index 1e5256e7561275b2fae3be5b5e1ff615ce145e38..d0730427ea241b6b46420cac4c13f6159465269f 100644
--- a/dist/build/webpack/loaders/next-flight-loader/index.js
+++ b/dist/build/webpack/loaders/next-flight-loader/index.js
@@ -47,7 +47,7 @@ async function transformSource(source, sourceMap) {
             }
             let esmSource = `\
 import { createProxy } from "${moduleProxy}"
-const proxy = createProxy("${this.resourcePath}")
+const proxy = createProxy(String.raw\`${this.resourcePath}\`)
 
 // Accessing the __esModule property and exporting $$typeof are required here.
 // The __esModule getter forces the proxy target to create the default export